added enemy hurt sound

Bullet-Not-Proof opened the pull request:
# Description

added enemy hurt sound (sound is a placeholder at the moment)

Fixes # (issue)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

using level climb code 'CLIMB' jump on the first slime that you see in the level the sound should play on the first and second hits but not the third. jumping on the cockroach in the room directly above will not play the sound as it has only one hitpoint

Bullet-Not-Proof committed:
added enemy hurt sound

Commit:
Compiled Games

Wills2022 reviewed (approved):
Looks good. Try to avoid adding or removing spaces when modifying code as this creates a lot of extra changes and makes the code a lot harder to review. Otherwise great work!

Bullet-Not-Proof committed:
ADDED: the ability for the player to cast a crude
shadow when behind a pipe
ADDED: ability for decorative objects to tell if
a player is intersecting it
ISSUES: the purpose of #include "types.h" in
block.h

Bullet-Not-Proof committed:
Merge branch 'main' of https://github.com/Bullet-Not-Proof/arcade-games

Bullet-Not-Proof committed:
Removed Compiled games
